types/utils.go

```diff
@@ -142,6 +143,8 @@ func GetRandomSignedHeaderWith(height uint64, dataHash header.Hash) (*SignedHead
 	signedHeader.Header.BaseHeader.Height = height
 	signedHeader.Header.DataHash = dataHash
 	signedHeader.Header.ProposerAddress = valSet.Proposer.Address
+	signedHeader.Header.ValidatorHash = valSet.Hash()
+	signedHeader.Header.BaseHeader.Time = uint64(time.Now().Unix()) + height*10
```

Comment:
Should be `UnixNano()`. See https://github.com/decentrio/rollkit/pull/1

Reply:
fixed by merge https://github.com/decentrio/rollkit/pull/1